CMakeLists.txt

```diff
@@ -65,10 +65,10 @@ add_library(roboteam_utils STATIC
         include/roboteam_utils/Grid.h
         src/Shadow.cpp
         include/roboteam_utils/Shadow.h
+        include/roboteam_utils/containers/circular_buffer.h
```

Comment:
still convinced headers dont have to be here but w/e

Reply:
No they don't need to be there, but it sometimes helps with CLion detecting that the file is within the project so that autocomplete etc. is on. It doesn't matter a bit for CMake